packages/server/src/question/question.entity.ts

```diff
@@ -45,7 +45,7 @@ export class QuestionModel extends BaseEntity {
   @Exclude()
   taHelpedId: number;
 
-  @CreateDateColumn()
+  @Column()
```

Comment:
What's the reason for this?

Reply:
CreateDateColumn wasn't correctly storing the createdAt date, it did something funky with the UTC conversion and actually messed up our times. I changed it to a normal Column and pass in `new Date()` when creating a question which works fine, but not sure if we should investigate this further.